src/sql/error_message.py

```diff
@@ -0,0 +1,38 @@
+import sqlglot
+
+SYNTAX_ERROR = "\nLooks like there is a syntax error " "in your query\n"
+
+
+def detail(msg, query=None):
+    msg = str(msg)
+    return_msg = SYNTAX_ERROR
+    if "syntax error" in msg:
+        query_list = query.split(";")
```

Comment:
usually it's at the end of query. can u give an example of `;` in between query?

Reply:
It should split this into 2 different queries which is what we want. because we need to parse each query through sqlglot

Reply:
We decided to parse using sqlglot only if there is an exception (otherwise there's no other better way to detect the syntax error). We wont be parsing using sqlglot initially to prevent false cases.

https://github.com/ploomber/jupysql/pull/400#issuecomment-1515621581

Reply:
So if there's an exception we do this:
1. User may write multiple `;` separated queries, so split and extract each query separately.
2. Parse each query through sqlglot and append this the error message.
3. Finally raise `UsageError` with the above message

Reply:
https://github.com/ploomber/jupysql/pull/400#issuecomment-1515038679